src/components/TitledList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Text, HStack, VStack, Divider, OrderedList, ListItem, Box } from '@chakra-ui/react';
+
+interface Props {
+  title: string;
+  listElements: string[] | undefined;
+}
+
+function TitledList({ title, listElements }: Props): JSX.Element {
+  return (
+    <VStack align="stretch" spacing={4}>
+      <HStack align="center" spacing={4}>
+        <Text fontWeight="bold" fontSize="xl">
+          {title}
+        </Text>
+        <Divider width="full" borderColor="green.400" />
```

Comment:
```suggestion
        <Divider width="full" borderColor="green" />
```
should fix the color issue